css/style.css

```diff
@@ -0,0 +1,225 @@
+/*  Add your styling here */
+/*body {
+  display: flex;
+  flex-direction: column;
+}
+*/
+header {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-around;
+  align-content: center;
+}
+
+.image-frame {
+  max-width: 50%;
+  margin-left: 5%;
+}
+
+.img-header {
+  width: 100%;
+  max-width: 200px;
+  margin-top: 2%;
+}
+
+.right-header {
+  display: flex;
+  flex-direction: column;
+  justify-content: space-around;
+}
+
+.menu-frame {
+  display: flex;
+  flex-direction: row-reverse;
+  justify-content: space-between;
+  margin-right: 4%;
+}
+
+.menu-header {
+  width: 15%;
+}
+
+.header-text {
+  font-family: "Gill Sans", "Gill Sans MT", Calibri, "Trebuchet MS", sans-serif;
+  font-size: 1.2rem;
+  text-align: center;
+  color: #457b9d;
+  padding-left: 25%;
+}
+
+.the-nav {
+  display: none;
+}
+
+main {
+  display: flex;
+  flex-direction: column;
+  justify-content: space-around;
+  align-content: center;
+}
+
+.welcome {
+  margin: 2%;
+  padding-left: 6%;
+  font-family: Arial, Helvetica, sans-serif;
+  font-size: 1.1rem;
+}
+
+.main-text {
+  margin: 2%;
+  margin-top: 0px;
+  padding-left: 4%;
+  padding-right: 4%;
+  font-family: Arial, Helvetica, sans-serif;
+  text-align: left;
+  font-size: 0.8rem;
+}
+
+.div-main-img {
+  max-width: 90%;
+}
+.main-image {
+  width: 100%;
+  margin-left: 10%;
+}
+
+.photos-sec {
+  display: flex;
+  flex-direction: column;
+  justify-content: space-around;
+  margin: 2%;
+  padding-left: 4%;
+  padding-right: 1%;
+}
+
+.div-photos {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-around;
+}
+
+.image1 {
+  margin: 2%;
+  width: 30%;
+}
+
+footer {
+  display: flex;
+  flex-direction: column;
+  justify-content: space-around;
+  margin: 4%;
+  background-color: #f1faee;
+  font-family: Georgia, "Times New Roman", Times, serif;
```

Comment:
I'd suggest you be more consistent with your font usage. It can be jarring to a user if the font is always changing